stylelint-polaris/tsconfig.json

```diff
@@ -5,7 +5,8 @@
     "outDir": "dist",
```

Comment:
Perhaps use `build` here to match polaris-react?

Reply:
https://github.com/Shopify/polaris/pull/5466#discussion_r846502024